### DISCUSSION TITLE
ImageGlass Moon 32-bit?

### DISCUSSION THREAD

**Discussion by ThomasLeigh** [d2phap/ImageGlass#1073] [General]
Hi,

It is interesting for me to experience the newest and greatest version of ImageGlass, i.e., the Moon releases. Unfortunately I use Windows 7 32-bit. Would You be willing to publish the Moon version also in 32-bit?

Kind regards,
Thomas

**Comment by d2phap** [comment]
Hi @ThomasLeigh 
64-bit IG uses different Magick.NET library (which is ~2x faster). Building 32-bit IG Moon requires extra work, that's why I only provide 64-bit IG Moon. I will consider to do it in the future.
